src/vos/vos_io.c

```diff
@@ -888,6 +888,7 @@ akey_fetch_single(daos_handle_t toh, const daos_epoch_range_t *epr,
 		goto out;
 
 	*rsize = rbund.rb_gsize;
+	ioc->ic_io_size += rbund.rb_gsize;
```

Comment:
you probably want to use rb_rsize, which is the local record size,  and rb_gsize is the sum of the rec size from all shards of ec obj.